docs/02.API-REFERENCE.md

```diff
@@ -8176,6 +8176,11 @@ main (void)
 
 Creates a `jerry_value_t` representing an undefined value.
 
+*Notes*:
+- Calling [jerry_release_value](#jerry_release_value) is optional.
+- Calling [jerry_acquire_value](#jerry_acquire_value) is optional.
```

Comment:
I would write something like this:
The undefined value is a fixed constant. Its value can be copied any number of times without calling `jerry_acquire_value`, and freeing it with `jerry_release_value` is optional. 

This is also true for true/false/null values, so add this comment to them.